### ISSUE TITLE
[ACCESSIBILITY] Assistant - Ask the Assistant screen has some unlabeled buttons (TalkBack)

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4196]
**Describe the issue**
Assistant screen is missing labels for:

- Thumbs up
- Thumbs down
- Send/Submit entry
- Speech-to-text microphone

**To Reproduce**
**Steps to reproduce the behavior:**
1. Using TalkBack
2. Go to 'Assistant'
3. Gesture until you hear 'Edit box, double tap to edit text'
4. Gesture right one more time and hear "Button, double tap to activate" **(This is for the voice entry microphone icon, does not give the user any context that this is a speech-to-text entry method.)**
5. Double tap, enter any question (for this I used 'Where is the nearest dining hall?' via voice entry button on device keyboard)
6. Gesture right one more time and hear "Cleared text, keyboard hidden, button, double tap to activate." when focus lands on microphone again instead of telling user Assistant is processing" (BUG# _____)
7. See error

**Expected behavior**
A clear and concise description of what you **_expect_** to happen.

**Actual behavior**
A clear and concise description of what **_actually_** happens.

**Screenshots**
![unlabeled buttons2](https://github.com/rokwire/illinois-app/assets/82892873/dadf0a29-8541-44ef-830d-bd0ecf705ec5)
![unlabeled buttons](https://github.com/rokwire/illinois-app/assets/82892873/35295887-20f6-4473-9ef1-e07a9fd5269e)

**Testing Environment & Devices (please complete the following information):**
• OS: Android Version 13
• Device: Google Pixel 4a

**Additional context**

**Comment by TodorBachvarov** [comment]
@acovher2, the button labelling is improved in version 6.0.21

**Comment by acovher2** [comment]
Verified as fixed in 6.0.36 android Prod
